src/core/migrations/0039_auto_20200515_1652.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-05-15 15:52
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+OLD_VALUE = "Dear {{ review_assignment.reviewer.full_name }}, <br><br>Thank you for agreeing to review \"{{ article.title }}\" in {{ article.journal.name }}. <br><br>You can now access the manuscript and the review process at: <a href=\"{% journal_url 'do_review' review_assignment.id %}\">{% journal_url 'do_review' review_assignment.id %}</a>. <br><br>Regards, <br>{{ request.user.signature|safe }}"
```

Comment:
This won't fix the bad url for modified templates, should we regex replace the bad url for the new one instead?